src/Nethermind/Nethermind.Evm.Test/Tracing/GethLike4byteTracerTests.cs

```diff
@@ -0,0 +1,82 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Collections.Generic;
+using FluentAssertions;
+using Nethermind.Core.Test.Builders;
+using Nethermind.Evm.Precompiles;
+using Nethermind.Evm.Tracing.GethStyle;
+using Nethermind.Evm.Tracing.GethStyle.Custom.Native.Tracers;
+using NUnit.Framework;
+namespace Nethermind.Evm.Test.Tracing;
+
+[TestFixture]
+public class GethLike4byteTracerTests : GethLikeNativeTracerTestsBase
+{
+    [Test]
+    public void Trace_call_input()
+    {
+        byte[] callInput = Prepare.EvmCode
+            .PushData(SampleHexData2)
+            .STOP()
+            .Done;
+
+        byte[] empty6ByteInput = new byte[6];
+
+        byte[] code = Prepare.EvmCode
+            .CallWithInput(TestItem.AddressA, 50000, callInput)
```

Comment:
Note to reviewers - I still need to add test cases for `DELEGATECALL`, `STATICCALL`, and `CALLCODE`

Reply:
will that test any different code-paths? If not I think we can skip it.